lib/hal/video.c

```diff
@@ -22,6 +22,9 @@
 #define NV_PMC_INTR_EN_0_INTA_HARDWARE 0x00000001
 #define NV_PMC_INTR_EN_0_INTA_SOFTWARE 0x00000002
 
+#define NV_USER_DAC_WRITE_MODE_ADDRESS 0x006813C8
```

Comment:
Ideally these go in some `.h` file, but ok for now.

Reply:
Agreed, but this is how hal/video.c does it.